src/app/profile/page.tsx

```
@@ -36,6 +38,37 @@ const Profile = () => {
+	const handleFormSubmit = async (
+		formData: UserProfile,
+		imageData: string | null
+	) => {
+		setIsSubmitting(true);
+		try {
+			const updatedProfile = { ...formData };
+			if (imageData) {
+				updatedProfile.photoURL = imageData;
+			}
+
+			const docRef = doc(db, 'users', uid!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)